tests/test_uds_auxiliary.py

```diff
@@ -317,3 +318,15 @@ def test_Response_wrap(self, uds_raw_aux_inst, mock_uds_config, mocker):
 
         assert resp.is_negative is True
         assert resp.nrc == NegativeResponseCode.CONDITIONS_NOT_CORRECT
+
+    def test_tester_present_sender(self, uds_raw_aux_inst, mocker):
```

Comment:
Another small unit test where you manipulate the thread in the context manager would be nice, for example:
```
        with uds_raw_aux_inst.tester_present_sender(0.01) as sender:
            sleep(0.1)
            sender.stop()
            assert not sender.is_alive()
            
        assert not sender.is_alive()
 ```

Reply:
For the reason discussed, this won't be possible, now the thread can be stopped only by exiting the context manager